js/components/Table/Row.js

```diff
@@ -4,29 +4,21 @@ import { map } from 'lodash';
 class Row extends React.Component {
   constructor() {
     super();
-
-    this.createRow = this.createRow.bind(this);
   }
 
-  createRow(data) {
-    const row = [
-      {
-        title: data.rank.toString().concat('. ', data.user.data.first_name),
```

Comment:
Is the rank added somewhere else?

Reply:
@chloealee arghhhhh oops this was a rebase mistake!

Reply:
@chloealee ranks are back!

Reply:
The photo above is updated too 📸